lnbits/app.py

```diff
@@ -285,17 +285,6 @@ async def restore_installed_extension(app: FastAPI, ext: InstallableExtension):
         ext.nofiy_upgrade()
 
 
-def register_routes(app: FastAPI) -> None:
```

Comment:
- why remove this method?
- it is better to have many small methods with well defined names rather than bigger methods which are hard to follow

Reply:
in this case i absolutely disagree, register_routes mixing up registering extension routes with initializing the core routers.

helper function for helper functions are hard to follow because you jump around everywhere in the code and makes it so you cant follow it!

it is a balance and it this case it went 1 abstraction to far